tools/DevDiagnostics/DevHome.DevDiagnostics/Pages/SystemResourceUsagePage.xaml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Page
+    x:Class="DevHome.DevDiagnostics.Pages.SystemResourceUsagePage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:models="using:DevHome.DevDiagnostics.Models"
+    mc:Ignorable="d"
+    NavigationCacheMode="Enabled">
+
+    <Page.Resources>
+        <models:HardwareTreeViewTemplateSelector x:Key="HardwareTreeViewTemplateSelector">
+            <models:HardwareTreeViewTemplateSelector.HardwareTemplate>
+                <DataTemplate x:DataType="models:HardwareNode">
+                    <TreeViewItem AutomationProperties.Name="{x:Bind Name}"
+                        Content="{x:Bind Name}"
+                        ItemsSource="{x:Bind Children}" />
+                </DataTemplate>
+            </models:HardwareTreeViewTemplateSelector.HardwareTemplate>
+            <models:HardwareTreeViewTemplateSelector.SensorGroupTemplate>
+                <DataTemplate x:DataType="models:SensorGroupNode">
+                    <TreeViewItem AutomationProperties.Name="{x:Bind Name}"
+                        Content="{x:Bind Name}"
+                        ItemsSource="{x:Bind Children}" />
+                </DataTemplate>
+            </models:HardwareTreeViewTemplateSelector.SensorGroupTemplate>
+            <models:HardwareTreeViewTemplateSelector.SensorTemplate>
+                <DataTemplate x:DataType="models:SensorNode">
+                    <TreeViewItem AutomationProperties.Name="{x:Bind Name}">
+                        <Grid>
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="200"/>

```

Comment:
Is this always going to be big enough, if the user has text size set to >100%?

Reply:
That's a good point. No it won't be big enough if the text size gets bigger. Is there a recommended way to set the width? I noticed that with text size increased, the column for current "cpu usage", "memory usage" is not big enough either

Reply:
Well, we could adjust the width dynamically using UISettings.TextScaleFactor as a multiplier - but that seems overkill. Alternatively, set TextWrapping=wrap, but that might look odd. Simplest would be to simply pick a much bigger width (say 300?), as we have lots of horizontal real estate to play with - not perfect, but likely would cover most cases.